SPECS/json-glib/json-glib.spec

```diff
@@ -91,6 +91,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_datadir}/installed-tests/*
 
 %changelog
+*   Fri Dec 04 2020 Andrew Phelps <anphel@microsoft.com> 1.4.4-6
+-   Use meson test in check section.
 *   Mon Jun 01 2020 Henry Beberman <henry.beberman@microsoft.com> 1.4.4-5
 -   Fix compilation issue with LDFLAGS and switch build to call meson directly.
 *   Sat May 09 00:20:47 PST 2020 Nick Samson <nisamson@microsoft.com> - 1.4.4-4
```

Comment:
Nit: since we're already here, could you remove the `00:20:47 PST ` part from Nick's entry?

Reply:
yes, will remove